src/main/asciidoc/http.adoc

```diff
@@ -1803,6 +1803,29 @@ use the full URL specified in the request URI:
 {@link examples.HTTPExamples#example60}
 ----
 
+=== Using HA PROXY protocol
+
+https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[HA PROXY protocol] provides a convenient way to safely transport connection
+information such as a client's address across multiple layers of NAT or TCP
+proxies.
+
+HA PROXY protocol can be enabled by setting the option {@link io.vertx.core.http.HttpServerOptions#useProxyProtocol(boolean)}
+and adding the following dependency in your classpath:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.netty</groupId>
+  <artifactId>netty-codec-haproxy</artifactId>
+  <version>4.1.18.Final</version>
```

Comment:
use `${maven.version}` instead

Reply:
maven.version is used for vertx dependencies not for netty

Reply:
ah right, then we should either use a comment here like <!-- Set netty version here --> or pass the netty version so we can resolve it using `${netty.version}`

Reply:
I could not find a way to pass netty.version provided from vertx-dependencies down to maven-processor-plugin

We either
1. Declare a new property and we will have to remember to copy the version in the new property and inside vertx-dependencies
2. Remove the version and add a note
3. Any other idea?

Reply:
let's go for 2, for now we can revisit later